modules/tftp/server.rb

```diff
@@ -63,6 +63,18 @@ def delete_file(file)
         logger.debug "TFTP: Skipping a request to delete a file which doesn't exists"
       end
     end
+
+    def delete_dir(dir)
+      if Dir.exist?(dir)
+        FileUtils.rm_rf dir
```

Comment:
`rm_rf` will happily ignore nonexisting directories. I'm not sure why `delete_file` has this wrapper.

I would be worried about someone passing in `..`, but haven't clearly audited the whole path. Perhaps that's also an issue for `delete_file`.

Reply:
Thanks for the hint! I made it more robust and secure.  The method is now called `delete_host_dir` and accepts a MAC address. The directory to delete is then constructed inside this method.